vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "genefilter"
```

Comment:
The title should be the same as in the PDF: "Using the genefilter function to filter genes from a microarray
dataset". See https://bioconductor.org/packages/release/bioc/vignettes/genefilter/inst/doc/howtogenefilter.pdf.